spec/spec_helper.rb

```diff
@@ -33,4 +33,6 @@
 RSpec.configure do |config|
   # Enable flags like --only-failures and --next-failure
   config.example_status_persistence_file_path = '.rspec_status'
+  config.filter_run focus: true
+  config.run_all_when_everything_filtered = true
```

Comment:
Dunno if y'all don't like this for some reason, would prefer it be added in a different PR or whatever. Easy to remove with a rebase.